File: CnpjScanner.Api/SampleCodes/sample.ts
const companyCnpj = "12.345.678/0001-90";
const unrelated = 123;
const taxId = 987654321;
let myVar: number = 123;
var legacy = 789;

class Business {
  public cnpjCode: string = "00.000.000/0000-00";
  private internalTaxNumber = 112233445566;
  public id: number = 123;
  private cnpjNumber = 12345678000190; // inferred as number
  protected taxId!: number;
  
  get taxGetter(): number {
    return 9988776655;
  }
}

interface ICompany {
  taxField: number;
  name: string;
}

enum TaxEnum {
  TaxValue = 12345678000190,
  OtherValue = 100
}

function registerCompany(cnpj: string, name: string, taxId: number) {
  const normalizedTax = taxId.toString();
  return normalizedTax;
}

File: CnpjScanner.Api/SampleCodes/sample.vb
Module Sample
    Const CNPJ_LENGTH As Integer = 14
    Dim companyCnpj = 12345678000190
    Dim unrelated = 123
    Dim CNPJ As Object
    Dim cnpj2 = "Company XYZ"
    Dim z As Integer

    Sub Register(taxId As Integer)
        Dim normalized = taxId.ToString()
    End Sub
End Module

Class Business
    Public Property TaxId As Integer
    Public Property TaxId As Integer = 12345678
    Private _internalCnpj As Integer = 12345678
    Private id As Integer
    Public count As Integer = 100
    Protected taxId = 12345678 ' Inferred type (VB.NET 2010+ allows this)

    Public Property ID As Integer
    Private _age As Integer

    Public Property Age As Integer
        Get
            Return _age
        End Get
        Set(value As Integer)
            _age = value
        End Set
    End Property
End Class

Interface ICompany
    Property CNPJ As String
    Property TaxId As Integer
End Interface

Enum TaxEnum
    CnpjValue = 12345678000190
    Other = 42
End Enum
